### PR TITLE
Gardening.

### PR DESCRIPTION
**Pull request by dan-zheng**
- Change all `@differentiable` operators constrained on `T: FloatingPoint`
  to be constrained on `T: TensorFlowFloatingPoint` for consistency.
- Omit return for single expression functions in Sources/TensorFlow/Operators/Math.swift for consistency.

### CODE REVIEW

File: Sources/TensorFlow/Operators/NN.swift [hunk 1]
```diff
@@ -76,36 +76,6 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
...
-    /// - Parameters:
-    ///     - axis: The batch dimension.
-    ///     - offset: The offset, also known as beta.
-    ///     - scale: The scale, also known as gamma.
-    ///     - epsilon: A small value added to the denominator for numerical stability.
-    @inlinable
-    @differentiable(
-        wrt: (self, offset, scale),
-        vjp: _vjpBatchNormalized where Scalar: TensorFlowFloatingPoint)
-    func batchNormalized(
```
Comment (dan-zheng) [inline Sources/TensorFlow/Operators/NN.swift]: Removing this is necessary because `Scalar` is constrained by `BinaryFloatingPoint`, not `TensorFlowFloatingPoint`.
```
/Users/danielzheng/swift-apis/Sources/TensorFlow/Operators/NN.swift:104:19: error: global function 'rsqrt' requires that 'Scalar' conform to 'TensorFlowFloatingPoint'
        let inv = rsqrt(variance + epsilon) * scale
                  ^
/Users/danielzheng/swift-apis/Sources/TensorFlow/Operators/Math.swift:543:13: note: where 'T' = 'Scalar'
public func rsqrt<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
            ^
```

This block is duplicated with `func batchNormalized` above for some reason. I took a look at `git blame` but couldn't figure out why.

Comment (rxwei) [reply Sources/TensorFlow/Operators/NN.swift]: I'm very surprised this was duplicated.
